app/models/start_teacher_training.rb

```diff
@@ -1,7 +1,7 @@
 class StartTeacherTraining < Base
   extend ApiOptions
 
-  attribute :initial_teacher_training_year_id, :string
+  attribute :initial_teacher_training_year_id, :integer
```

Comment:
Looks good, just the inital typo is causing both `"intital_teacher_training_year_id"=>"12917"` and `"initial_teacher_training_year_id"=>12917` to appear in the session